helm/prometheus-meta-operator/templates/prometheus-rules/prometheus.rules.yml

```diff
@@ -34,3 +34,33 @@ spec:
         severity: page
         team: atlas
         topic: observability
+    - alert: PrometheusHighMemoryUsage
+      annotations:
+        description: |-
+          {{`Prometheus in pod {{ $labels.namespace }}/{{ $labels.pod }} is using >95% of node's memory.
+          VALUE = {{ $value }}
+          LABALE = {{ $labels }}`}}
+        opsrecipe: TODO/
+      # `node` label is sometimes a name and sometimes IP, and it was different
+      # between those metrics on kvm installations, so extract IP to be able to match
+      expr: |
+        sum by (node_ip, pod) (
```

Comment:
The rule makes sense to me :)

Reply:
I'm wondering if we should not alert only if the memory is increasing after 95 % instead of when it's above.